### ISSUE TITLE
honour preexisting phase from pedigree or 10x Chromium

### ISSUE THREAD

**Issue by mdkeehan** [poruloh/Eagle#12]
A useful feature to have would be to honor pre-existing phase.
Typically sources of pre-existing phase are
1) pedigree relationships : duo's, trio's, half-sib sire families
2) 10x Chromium long ranger
3) GATK style read backed phasing from short read molecular information.
4) Information from an already phased snp-chip panel where some samples are on both
     accurate high density snp chip but also have low coverage whole genome sequence that is desired to be phased.  

One way to implement it would be to respect phase in the input VCF's and then extend it with 
the LD based information from eagle .

**Comment by poruloh** [comment]
I agree that this feature would be useful, and I tried to implement it in the --usePS flag last year (which is supposed to incorporate VCF phase constraints in FORMAT:PS). Unfortunately, I did not achieve an improvement in accuracy in the limited testing I did at the time, and I eventually heard that at least one user suspected it might have a bug. You are welcome to try --usePS if you would like; I would be interested in the results (although I can't promise I will have time to return to the issue anytime soon).
